tutorials/qdrant_langchain_instrumentation_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,6089 @@
+{
```

Comment:
<p>We need to remove this image since it's not using pinecone anymore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2440/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>